### PR TITLE
Add parents array when upserting Gdrive documents

### PR DESCRIPTION
**Pull request by lasryaric**

### CODE REVIEW

File: connectors/src/connectors/google_drive/temporal/activities.ts [hunk 1]
```diff
@@ -389,14 +390,20 @@ async function syncOneFile(
   });
 
   if (documentContent.length <= MAX_DOCUMENT_TXT_LEN) {
+    const parents = await getFileParentsMemoized(
```
Comment (philipperolet) [inline connectors/src/connectors/google_drive/temporal/activities.ts:393]: Does it return a parent list as per the specs of the parents field [here](https://github.com/dust-tt/dust/blob/main/core/src/data_sources/data_source.rs) line 93

File: connectors/src/connectors/google_drive/temporal/activities.ts [hunk 2]
```diff
@@ -389,14 +390,20 @@ async function syncOneFile(
...
+    );
     await upsertToDatasource({
       dataSourceConfig,
       documentId,
       documentText: documentContent,
       documentUrl: file.webViewLink,
       timestampMs: file.updatedAtMs,
       tags,
-      parents: [],
+      parents: parents.map((p) => p.id),
```
Comment (philipperolet) [inline connectors/src/connectors/google_drive/temporal/activities.ts:406]: importantly the first element of the array must be the document id itself; is it the case here?

**Comment by philipperolet** [comment]
linting errors

File: connectors/src/connectors/google_drive/temporal/activities.ts (hunk 2 above)
Comment (spolu) [inline connectors/src/connectors/google_drive/temporal/activities.ts:406]: You want to add the file itself as well (I thiiiiiink it's not in the return of that function, is it?)

**Review by philipperolet — CHANGES_REQUESTED** [review]
LGTM upon confimation of the questions I asked (+ linting errors)

This makes me want to cry, this is so nice and short. My PR for notion is awful :cry:

**Review by philipperolet — APPROVED** [review]
LGTM :+1:

**Pull request merged**